src/less/blog-v2/main.less

```diff
@@ -34,6 +35,26 @@ body {
   .shine-gradient;
 }
 
+// Author banner on author pages
+.author-banner {
+  background-color: white;
+  .box-shadow;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: @margin-sm;
+  margin-bottom: @margin-sm;
+  .-image {
+    width: 135px;
+    height: 135px;
+    border-radius: 50%;
```

Comment:
I think we might need an `object-fit: cover` in here too. Some of the images (I think especially if the original aspect ratio isn't a 1x1) look distorted.